.github/actions/performance/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
     - name: Discover `test/bench` changes
       if: ${{ inputs.is_fork == 'false' }}
       run: |
-        git diff --no-index $(nix build --print-out-paths .#tests.bench)/share test/bench/ok | patch -p1 -R
+        git diff --no-index $(nix build --print-out-paths .#tests.bench)/share test/bench/ok | patch -f -u -p1 -R
```

Comment:
This is because sometimes we got an exit code `1` that messed up the pipeline. `-f` fixes that for good.

Reply:
Only it doesn't. It is super strange.

Reply:
Applied a band-aid until I understand this: 587d1301f.